lib/active_shipping/carriers/usps.rb

```diff
@@ -113,10 +140,22 @@ class USPS < Carrier
       :package_service => 'PACKAGESERVICE'
     }
 
+    CONTENT_TYPES = Hash.new('MERCHANDISE').update(
+      :sample => 'SAMPLE',
+      :gift => 'GIFT',
+      :documents => 'DOCUMENTS',
+      :return => 'RETURN',
+      :humanitarian => 'HUMANITARIAN',
+      :dangerous_goods => 'DANGEROUSGOODS',
+      :cremated_remains => 'CrematedRemains',
+      :nonnegotiable_document => 'NonnegotiableDocument'
+    )
+
     ATTEMPTED_DELIVERY_CODES = %w(02 53 54 55 56 H0)
 
     # Array of U.S. possessions according to USPS: https://www.usps.com/ship/official-abbreviations.htm
     US_POSSESSIONS = %w(AS FM GU MH MP PW PR VI)
+    DOMESTIC_CODES = US_POSSESSIONS + ['US', nil]
```

Comment:
That shouldn't be necessary - rather than assume it's `US` when no country is set, we should raise on a missing country in a `Location`; when it comes to the location elements of shipping, a country is one of the most important factors.